tests/simulator/test_advance.py

```
@@ -212,188 +250,180 @@ def vthz(*xyz):
-                        if box.ndim == 1:
-                            failed_i = np.where(np.abs(slice1 - slice2) > 5.5e-15)
-
-                        if box.ndim == 2:
-                            failed_i, failed_j = np.where(
-                                np.abs(slice1 - slice2) > 5.5e-15
+                        failed_i, failed_j = np.where(np.abs(slice1 - slice2) > 5.5e-15)
+
+                        def makerec(lower, upper, dl, fc="none", ec="g", lw=1, ls="-"):
+                            origin = (lower[0] * dl[0], lower[1] * dl[1])
+                            sizex, sizey = [
+                                (u - l) * d for u, l, d in zip(upper, lower, dl)
```

Ruff (0.15.2)

[error] Ambiguous variable name: `l`

(E741)